### DISCUSSION TITLE
Position of model

### DISCUSSION THREAD

**Discussion by jgery07** [microsoft/MixedRealityToolkit-Unity#9067] [Q&A]
Hi all,
thank you very much for MRTK, it's very good,
I ask something, how to choose the place of the model in relation to reality?
Maybe the solution will be the use of QR code as a start place?
is it difficult to implement?

**Comment by polar-kev** [comment]
Hi @jgery07, sorry for the late reply. Here are a few topics for you to explore,
[Getting a coordinate system from a QR code](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/platform-capabilities-and-apis/qr-code-tracking#getting-the-coordinate-system-for-a-qr-code)
[World locking tools for Unity](https://microsoft.github.io/MixedReality-WorldLockingTools-Unity/README.html)
[Azure Object Anchors (in preview)](https://azure.microsoft.com/en-us/services/object-anchors/)
